dev/Materials/Acrylic/AcrylicBrush_rs2_themeresources.xaml

```diff
@@ -241,14 +241,14 @@
                 BackgroundSource="Backdrop"/>
             <local:AcrylicBrush
                 x:Key="AcrylicBackgroundFillColorBaseBrush"
-                TintColor="#1C1C1C"
-                TintOpacity="0.8"
+                TintColor="#202020"
+                TintOpacity="0.5"
```

Comment:
Leave these as TintOpacity="0.8". In rs2 acrylic didn't have TintLuminosityOpacity so the TintOpacity has to be more opaque.